selectrum.el

```diff
@@ -1249,6 +1249,16 @@ ignores the currently selected candidate, if one exists."
       selectrum--start-of-input-marker
       selectrum--end-of-input-marker))))
 
+(defvar selectrum--crm-separator-alist
+  '((":\\|,\\|\\s-" . ",")
+    ("[ \t]*:[ \t]*" . ":")
+    ("[ \t]*,[ \t]*" . ",")
+    (" " . " "))
```

Comment:
Somewhat scary but I suppose it doesn't have a great potential to spiral out of control :P 
Is there a way for the user to set the delimiter to be auto-inserted if they use a custom regexp? Just a thought -- I don't think it's a big problem if there isn't.

Reply:
If the value of `crm-separator` isn't one of this list the old approach is used where you have to input `crm-separator` manually so this shouldn't get us into trouble. If the user uses a custom regexp and they want to have the separator auto inserted they would have to add it to  `selectrum--crm-separator-alist`. We could make it a public option.

Reply:
Yeah, that is what I was thinking. We could also have a separate user option to override the auto-inserted separator, which would be more convenient for the user but on the other hand would contribute to the proliferation of user options.

Reply:
There are probably not to many cases where `crm-separator` isn't a member of this list maybe we can wait if someone complaints and then we can still add a user option.

Reply:
Sounds like a pragmatic idea to me!